tests/controllers/TalksControllerTest.php

```diff
@@ -12,6 +12,17 @@
 
 class TalksControllerTest extends TalkBase
 {
+    private $config;
+
+    public function setUp()
+    {
+        $this->config = [
+            'email' => [
+                'from' => 'source@example.com',
+            ],
+            'website_url' => 'http://example.com',
+        ];
+    }
```

Comment:
These changes are not related to the TypeHints, so I'd move them to a different PR

Reply:
This change is because the `ApiController`'s ctor has a type-hinted config as an array. So this change is essentially mocking the config so the test will pass.